templates/smartstart/index.html

```diff
@@ -0,0 +1,508 @@
+{% extends "smartstart/base_smartstart.html" %}
+
+
+{% block title %}Smart things as a service{% endblock %}
+{% block meta_description %}SMART START is a comprehensive toolbox that allows enterprises to bootstrap commercial IoT products. With a 2 week delivery, SMART START reduces the risks and uncertainty involved in launching and scaling commercial IoT products while accelerating time to market.{% endblock meta_description %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1H8JymT13-pHvPh2KXeXYaAH4HoPYWiwAQXXUDqlGzLM/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped is-dark">
+  <div class="row u-vertically-center">
+    <div class="col-7">
+      <h1 class="u-sv2">Smart things as a service</h1>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Bring your IoT devices to market in the blink of an eye with SMART START</li>
+        <li class="p-list__item is-ticked">Focus on your apps, we handle the rest</li>
+        <li class="p-list__item is-ticked">Fast to market, low risk, and scalable</li>
+      </ul>
+      <p>
+        Canonical offers full-service enablement, customisation and development to get your first device to market. App store and security updates guaranteed.
+      </p>
+      <p>
+        <a class="p-button--positive" href="/pricing/devices">Pricing</a>
+        <a href="/smart-start/guide">A guide to SMART START&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+    <div class="col-5 col-start-large-8 u-align--center u-hide--small">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/431b4ca2-ubuntu+core+18+circular+.svg",
+          alt="",
+          width="300",
+          height="300",
+          hi_def=True,
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-deep">
```

Comment:
I would probably try this without the `is-deep` to bring more content above the fold.